client/test/utils/mockery_utils.js

```diff
@@ -0,0 +1,38 @@
+const mockery = require('mockery');
+
+
+// Bunch of functions to make tests less verbosed.
+module.exports = {
+
+  // Register test dispatcher as a replacement of the official dispatcher.
+  initDispatcher: (dispatcher) => {
+    mockery.registerMock('../dispatcher/dispatcher', dispatcher);
+    mockery.registerMock(
+      '../libs/flux/dispatcher/dispatcher', dispatcher);
+  },
+
+  // Configure mockery to run properly with stores.
+  initForStores: () => {
+    mockery.enable({
+      warnOnUnregistered: true,
+      useCleanCache: true,
+    });
+
+    mockery.registerAllowables([
+      'node-event-emitter',
+      'immutable',
+      'lodash',
+      '../constants/app_constants',
+      '../libs/flux/store/store',
+      '../app/stores/account_store',
+      '../app/stores/layout_store',
```

Comment:
I think the _allowables_ are dependents of the module we are currenlty testing (i.e. I will probbly not allow the sames modules when I test the _AccountsStore_ and the _MessagesActionsCreator_). Maybe we can register the commons here, but not the app specifics modules.